src/App.js

```diff
@@ -1,86 +1,84 @@
+    setLikeCount([...likeCount, 0]);
   }
 
   const switchInput = () => {
     setIsSingleView(!isSingleView);
   }
 
-  const nextFood = () => {
-    if (currIndex < foodList.length - 1) {
-      setCurrIndex(currIndex + 1);
-    } else {
-      setCurrIndex(0);
-    }
-  }
-
-
-  const [likes, setLikes] = useState([]);
   // true = single view?
   return (
     <div className="App">
       <header className="App-header">
         <div>
           {
-            <button onClick={switchInput}> {
+            <button className="float-switch-button my-button" onClick={switchInput}> {
               isSingleView ? "View List" : "View Single"
             } </button>
           }
-          <h2> Add a New Food</h2>
-          <input type="text" onChange={(event) => { setInputFood(event.target.value) }} value={inputFood} />
-          <div>
-            <button onClick={addFood}>  Submit </button>
-          </div>
-        </div>
-        <div>
-          {isSingleView && (
+          <div className="grid-container">
             <div>
-              <Food
-                name={foodList[currIndex]}
-                index={currIndex}
-                foodList={foodList}
-                setFoodList={setFoodList}
-              >
-              </Food>
-              <button onClick={nextFood}> Pass Me </button>
+              <h2 id="main-title"> Add a New Food</h2>
+              <input className="input-field" type="text" onChange={(event) => { setInputFood(event.target.value) }} value={inputFood} />
+              <div>
+                <button className="my-button" onClick={addFood}>  Submit </button>
+              </div>
             </div>
-          )
-          }
-          {!isSingleView &&
-            foodList.map((value, index) => {
-              return <Food
-                name={value}
-                index={index}
-                foodList={foodList}
-                setFoodList={setFoodList}
-                likes={likes}
-                setLikes={setLikes}
-              >
-              </Food>
-            })
-          }
+
+            <div>
+              { foodList.length < 1 &&
+                <h4 className="text-colorset"> Please enter a food! </h4>
+              }
+              {isSingleView && foodList.length > 0 &&  (
+                <div className="food-item">
+                  <Food
+                    name={foodList[currIndex]}
+                    index={currIndex}
+                    foodList={foodList}
+                    setFoodList={setFoodList}
+                    likeCount={likeCount}
+                    setLikeCount={setLikeCount}
+                    setCurrIndex={setCurrIndex}
+                    isSingleView={isSingleView}
+                  >
+                  </Food>
+                </div>
+              )
+              }
+              {!isSingleView && foodList.length > 0 && 
+                foodList.map((value, index) => {
+                  return  <div className="food-item"> <Food
```

Comment:
probably need to add in a key! see [here](https://github.com/uclaacm/teach-la-dev-training-s21/tree/main/react%2Bmisc/3_review_react%2Breact_structure#list-and-keys) for how to fix and more details
![image](https://user-images.githubusercontent.com/29292203/117764457-5c589800-b1e1-11eb-9064-854e5874ac9d.png)

and [here](https://reactjs.org/docs/lists-and-keys.html#keys) for the react docs about keys!